src/perf/lib/RpsClient.cpp

```diff
@@ -24,6 +24,8 @@ PrintHelp(
         "RPS Client options:\n"
         "\n"
         "  -target:<####>              The target server to connect to.\n"
+        "  -bind:<addr>                Local IP address(es) to bind to.\n"
```

Comment:
>Local IP address(es) to bind to [](http://example.com/codeflow?start=39&length=31)

only IP address or it could be IP:port? Is this the change that you used to make secnetperf generate consistent perf on lab machines? If so, you bound to the same set of ports, right?

Reply:
Yes, you can use a port. `1.1.1.1:8000,1.1.1.2:8001`

Reply:
the text should be changed to reflect that.